packages/eslint-config-sheriff/src/getAstroConfig.ts

```diff
@@ -37,23 +37,18 @@ export const getAstroConfig = (
         },
       },
       settings: {
-        // 'import/core-modules': [
-        //   'astro:actions',
-        //   'astro:assets',
-        //   'astro:db',
-        //   'astro:content',
-        //   'astro:container',
-        //   'astro:env',
-        //   'astro:i18n',
-        //   'astro:middleware',
-        //   'astro:transitions',
-        //   'astro:transitions/client',
-        // ],
-        'import/parsers': {
-          'astro-eslint-parser': ['.astro'],
-          '@typescript-eslint/parser': ['.ts', '.tsx', 'mts', 'cts'],
-          espree: ['.js'],
-        },
+        'import-x/core-modules': [
+          'astro:actions',
+          'astro:assets',
+          'astro:db',
+          'astro:content',
+          'astro:container',
+          'astro:env',
+          'astro:i18n',
+          'astro:middleware',
+          'astro:transitions',
+          'astro:transitions/client',
+        ],
```

Comment:
these were commented, so they should stay commented. Uncommenting them is a unrelated change in this PR.